exclude_files.txt

```diff
@@ -1,17 +1,20 @@
 .git*
-.pytest_cache/
-*.egg-info/
-.vscode/
+*.pyc
```

Comment:
I changed the tarball command to be the script provided by phantom instead of `tar`.  The name `exclude_files.txt` is hard-coded into phantom as the name of the file to read from when looking from exclusions. It also doesn't seem to treat slashes the way we would expect, instead any folder or file that matches the pattern is what gets ignored